data/human/culture conversations.txt

```diff
@@ -120,6 +120,35 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Class Trip to Port"
+	minor
+	source
+		attributes "south"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`A group of children wave at you as you step off your ship, marveling at the ships on display as their teachers try and fail to hold their attention. You could see the fading of their uniform colors from afar as their group walked past, yet none of them looked one bit displeased. You wonder how many of them may eventually own their own spaceship and fly across the stars as you make your way into the spaceport itself.`
```

Comment:
```suggestion
			`A group of children wave at you as you make your way to the port, marveling at the ships on display as their teachers try and fail to hold their attention. You could see the fading of their uniform colors from afar as their group walked past, yet none of them looked one bit displeased. You wonder how many of them may eventually own their own spaceship and fly across the stars as you make your way into the spaceport itself.`
```

Reply:
Then it no longer makes sense as an interruption between disembarking, and the last line going to the port.